config-windows.md

```diff
@@ -0,0 +1,99 @@
+# Windows-specific Container Configuration
+
+This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
+
+## Resources
+
+You can configure a container's resource limits via the OPTIONAL `resources` field of the Windows configuration.
+
+### Memory
+
+`memory` is an OPTIONAL configuration for the container's memory usage.
+
+The following parameters can be specified:
+
+* **`limit`** *(uint64, optional)* - sets limit of memory usage in bytes.
+
+* **`reservation`** *(uint64, optional)* - sets the guaranteed minimum amount of memory for a container in bytes.
+
+#### Example
+
+```json
+    "windows": {
+        "resources": {
+            "memory": {
+                "limit": 2097152,
+                "reservation": 524288
+            }
+        }
+    }
+```
+
+### CPU
+
+`cpu` is an OPTIONAL configuration for the container's CPU usage.
+
+The following parameters can be specified:
+
+* **`count`** *(uint64, optional)* - specifies the number of CPUs available to the container.
```

Comment:
I'm fine with `uint64` here, but am enjoying a mental image of a containers that needs 4.3 billion CPUs ;).